scss/layout-about/_location.scss

```diff
@@ -0,0 +1,66 @@
+@use '../utilities' as *;
```

Comment:
- [ ] **You have used the same CSS declaration multiple times throughout your stylesheets. It is advisable to extract them in their own classes. Please fix this.**

Using the same declaration, again and again, violates the principle of dry. To keep your codebase concise, you can extract them into classes. For instance, declarations like `display: flex`, `text-align: center`, `align-items: center`, etc, will have static values. Rather than using them multiples times, you can extract them in their own classes and apply those classes to your HTML element. This will save you a lot of extra work.
 You can do something as follows. 
```
.d-flex {
  display: flex
}
.text-center {
  text-align: center
}
```